### PR TITLE
Added blog post about PSDevCon 2023 and translations

### PR DESCRIPTION
**Pull request by nicosomb**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Added blog post about PSDevCon 2023 and translations

### CODE REVIEW

File: content/news/posts/2024/do-you-speak-prestashop.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+At PrestaShop, we use it in many cases:
+
+- Updates of internal dependencies (PrestaShop modules)
+- In pull requests: nearly 40 checks are launched via Github Actions (tests, pull request validation, JS or PHP linter, UI or integration tests, etc.)
+- Nightly
+- Builds of our Docker images
+- Update and deployment of our “Top contributors” and “Top translators” sites
+
+And of course ... for translation management!
```
Comment (coffrep) [inline content/news/posts/2024/do-you-speak-prestashop.md]:
```suggestion
And of course... for translation management!
```

File: content/news/posts/2024/do-you-speak-prestashop.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
+---
+layout: post
+title: "PSDevCon 2023: Do you speak PrestaShop"
```
Comment (coffrep) [inline content/news/posts/2024/do-you-speak-prestashop.md:3]:
```suggestion
title: "PrestaShop DevCon 2023: Do you speak PrestaShop"
```

**Review by coffrep — APPROVED** [review]
Hello, great post thanks for bringing it to the blog, glad to see it ready for publishing! Just added two modifications, one of them being in the title. Let me know if this works for you. I'm approving on my end with changes.

File: content/news/posts/2024/do-you-speak-prestashop.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+subtitle: "How the community works to translate the software into multiple languages."
+date:   2024-02-05
```
Comment (kpodemski) [inline content/news/posts/2024/do-you-speak-prestashop.md]:
```suggestion
date:   2024-02-06
```

File: content/news/posts/2024/do-you-speak-prestashop.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
+---
+layout: post
+title: "PrestaShop DevCon 2023: Do you speak PrestaShop"
```
Comment (kpodemski) [inline content/news/posts/2024/do-you-speak-prestashop.md:3]:
```suggestion
title: "PrestaShop DevCon 2023: Do you speak PrestaShop?"
```

File: content/news/posts/2024/do-you-speak-prestashop.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+subtitle: "How the community works to translate the software into multiple languages."
+date:   2024-02-05
+authors: [ JulieVarisellaz, NicolasLoeuillet ]
+icon: icon-leaf
+tags: [ news, event, community ]
+---
+
+## Do you speak PrestaShop?
+
+On November 30, Nicolas Lœuillet and Julie Varisellaz gave a presentation about the project’s translations at the first-ever PrestaShop Developer Conference.
```
Comment (kpodemski) [inline content/news/posts/2024/do-you-speak-prestashop.md:13]:
```suggestion
On November 30th, Nicolas Lœuillet and Julie Varisellaz gave a presentation about the project’s translations at the first-ever PrestaShop Developer Conference.
```

File: content/news/posts/2024/do-you-speak-prestashop.md [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+A string must be validated by a proofreader to be sent to PrestaShop. Proofreaders are translators with validation rights. Their role is to ensure translations’ quality.
+
+Once a proofreader has translated and validated a string, it is sent to PrestaShop stores. Synchronization takes place every day. You can retrieve the latest translations from your store. To do this, don't forget to update the language in your back office (In the International menu > Translations > Update a language). Translations you have customized won’t be overwritten.
+
+## Contributions 
+
+Coding isn't the only way to contribute to an open source project. Translation is another way to contribute!
+Join the translator community [on Crowdin](https://crowdin.com/project/prestashop-official). Either your language is already 100% translated and you can suggest improvements or the language is not yet 100% translated and you can contribute to help achieve this goal.
+
+If you're interested, you can check our monthly blog post on the blog. We also have a [Slack channel]([prestashop.slack.com](http://prestashop.slack.com/)) dedicated to translations.
```
Comment (kpodemski) [inline content/news/posts/2024/do-you-speak-prestashop.md]:
```suggestion
If you're interested, you can check our monthly blog post on the blog. We also have a [Slack channel](https://www.prestashop-project.org/slack/) dedicated to translations.
```

File: content/news/posts/2024/do-you-speak-prestashop.md [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+6. Create a pull request manually
+7. Get feedback from the Content team in the pull request’s comment section
+8. Apply feedback locally in PrestaShop
+9. Go to step 4 again
+
+In the end, it was really hard work and time-consuming.
+
+## How we handle translations now
+
+Thanks to GitHub actions!
```
Comment (kpodemski) [inline content/news/posts/2024/do-you-speak-prestashop.md]:
```suggestion
Thanks to GitHub actions, we automated many of the steps we mentioned above!
```

**Comment by kpodemski** [comment]
Hi @Julievrz @nicosomb 

I added a few suggestions, let me know what you think, we'll publish it right after.

File: content/news/posts/2024/do-you-speak-prestashop.md (hunk 4 above)
Comment (Julievrz) [inline content/news/posts/2024/do-you-speak-prestashop.md]: I would even say "PrestaShop Developer Conference 2023: Do you speak PrestaShop?"

File: content/news/posts/2024/do-you-speak-prestashop.md (hunk 5 above)
Comment (Julievrz) [reply content/news/posts/2024/do-you-speak-prestashop.md:13]: "On November 30" is correct. See Grammarly's article: 
Screenshot 2024-02-06 at 14 22 57
https://www.grammarly.com/blog/how-to-write-dates/#:~:text=Many%20people%20get%20confused%20about,13th%2C%202007%20Daniel%20was%20born.

File: content/news/posts/2024/do-you-speak-prestashop.md (hunk 4 above)
Comment (kpodemski) [reply content/news/posts/2024/do-you-speak-prestashop.md]: we used `PSDevCon 2023:` in the previous article from the conference, we probably should keep the same format

Comment (coffrep) [reply content/news/posts/2024/do-you-speak-prestashop.md:3]: As suggested by @Julievrz I made an edit.

Comment (Julievrz) [reply content/news/posts/2024/do-you-speak-prestashop.md]: Ok then :)

Comment (kpodemski) [inline content/news/posts/2024/do-you-speak-prestashop.md]:
```suggestion
title: "PSDevCon 2023: Do you speak PrestaShop"
```

**Comment by kpodemski** [comment]
thanks @Julievrz @nicosomb @coffrep

**Pull request merged**